UI/frontend-plugins/frontend-tools/scripts.cpp

```diff
@@ -219,6 +220,20 @@ ScriptsTool::~ScriptsTool()
 		       ui->scripts->currentRow());
 }
 
+void ScriptsTool::showPythonVersion()
+{
+	bool loaded = obs_scripting_python_loaded();
+	QString label;
+	if (loaded) {
```

Comment:
Minor nitpick - `loaded` isn't used anywhere outside of this branch, so could possibly suffice to use `if(obs_scripting_python_loaded())`.

Reply:
Yeah I did this in prep for a future capability, but better to keep it simple for now. Fixed.